addon/components/sequential-render.js

```diff
@@ -258,6 +258,8 @@ export default Component.extend({
         quickRender
       } = getProperties(this, 'renderPriority', 'taskName', 'quickRender');
 
+      delete get(this, 'renderStates').scheduledCalls[taskName];
```

Comment:
Can we consider exposing methods that add / remove the property instead of directly mutating the property?

Reply:
Have addressed in the latest commit